src/module/datasworn.ts

```diff
@@ -114,132 +174,125 @@ export async function importFromDatasworn() {
   })
 
   // Domains
   const domainsJson = await fetch(
     'systems/foundry-ironsworn/assets/delve-domains.json'
   ).then((x) => x.json())
   const domainsToCreate = domainsJson.Domains.map((rawDomain) => {
+    const _id = DOMAIN_IDS[rawDomain.Name]
     const domainData = {
+      _id,
       type: 'delve-domain',
       name: rawDomain.Name,
       img: DOMAIN_IMAGES[rawDomain.Name],
-      data: {
+      system: {
         summary: rawDomain.Summary,
         description: rawDomain.Description,
-        features: [] as any[],
-        dangers: [] as any[],
+        features: importDelveFeaturesOrDangers(
+          rawDomain.Features,
+          'feature',
+          _id,
+          21
+        ),
+        dangers: importDelveFeaturesOrDangers(
+          rawDomain.Dangers,
+          'danger',
+          _id,
+          31
+        ),
       },
     }
 
-    let low = 21
-    for (const feature of rawDomain.Features) {
-      domainData.data.features.push({
-        low,
-        high: feature.Chance,
-        description: feature.Description,
-      })
-      low = feature.Chance + 1
-    }
-    low = 31
-    for (const danger of rawDomain.Dangers) {
-      domainData.data.dangers.push({
-        low,
-        high: danger.Chance,
-        description: danger.Description,
-      })
-      low = danger.Chance + 1
-    }
-
     return domainData
   })
   await Item.createDocuments(domainsToCreate, {
     pack: 'foundry-ironsworn.ironsworndelvedomains',
+    keepId: true,
   })
 
   // Foes
-  const foesJson = await fetch(
-    'systems/foundry-ironsworn/assets/foes.json'
-  ).then((x) => x.json())
-  const foesToCreate = [] as (ItemDataConstructorData &
-    Record<string, unknown>)[]
-  for (const category of foesJson.Categories) {
-    for (const foe of category.Foes) {
-      const description = await renderTemplate(
-        'systems/foundry-ironsworn/templates/item/foe.hbs',
-        {
-          ...foe,
-          Category: category.Name,
-          CategoryDescription: category.Description,
-        }
-      )
-
-      foesToCreate.push({
-        type: 'progress',
-        name: foe.Name,
-        img: FOE_IMAGES[foe.Name] || undefined,
-        data: {
-          description,
-          rank: foe.Rank.toLowerCase(),
-        },
-      })
-    }
-  }
-  await Item.createDocuments(foesToCreate, {
-    pack: 'foundry-ironsworn.ironswornfoes',
-  })
+  // const foesJson = await fetch(
+  //   'systems/foundry-ironsworn/assets/foes.json'
+  // ).then((x) => x.json())
+  // const foesToCreate = [] as (ItemDataConstructorData &
+  //   Record<string, unknown>)[]
+  // for (const category of foesJson.Categories) {
+  //   for (const foe of category.Foes) {
+  //     const description = await renderTemplate(
+  //       'systems/foundry-ironsworn/templates/item/foe.hbs',
+  //       {
+  //         ...foe,
+  //         Category: category.Name,
+  //         CategoryDescription: category.Description,
+  //       }
+  //     )
```

Comment:
May as well delete all this commented-out code, we've moved fully to Dataforged for these things.